Module_02/README.md

```diff
@@ -1,5 +1,196 @@
-# Working in the Command Line
+# Git & Github
 
-## Linux 101
+### Before the Lesson
+
+Please go through the following steps before the lesson if possible. We can troubleshoot at the start of the lesson if you run into issues.
+
+**Install Xcode**
+
+Please, please, please do this before the lesson. Everything else takes under a minute, but this step takes awhile to run.
+
+https://developer.apple.com/xcode/
+
+**Open a Terminal**
+
+You'll be running the following commands in the terminal. You should have covered installing/adding in the module on command line and bash. If you didn't, there is a terminal application that comes pre-installed on Macs which works fine, but I highly recommend this terminal replacement:
```

Comment:
maybe say module 0 or add a link to the first module, in the sentence `You should have covered installing/adding in the module on command line and bash`